roles/custom/matrix-bot-draupnir/templates/production.yaml.j2

```diff
@@ -56,8 +67,8 @@ managementRoom: {{ matrix_bot_draupnir_management_room | to_json }}
 # Deprecated and will be removed in a future version.
 # Running with verboseLogging is unsupported.
 # Whether Draupnir should log a lot more messages in the room,
-# mainly involves "all-OK" messages, and debugging messages for when Draupnir checks bans in a room.
-#verboseLogging: false
+# mainly involves "all-OK" messages, and debugging messages for when draupnir checks bans in a room.
+verboseLogging: false
```

Comment:
Is this intentional?

Reply:
Its a change that is irrelevant in the end as its the default behavior and yes i didnt see a reason to comment it out when its in the upstream config file like that. 

I did confirm with Gnuxie that its still a needed config flag as in i couldnt just remove it from the codebase like was done to other config flags that went away as part of this PR.